src/components/app/app.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import Display from '../display';
+import ButtonDisplay from '../button-panel';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: '0',
+    };
+  }
+
+  render() {
+    const { value } = this.state;
+    return (
+      <main>
+        <Display value={value} />
+        <ButtonDisplay />
+      </main>
```

Comment:
i think requirement says wrap in `<div>` tags instead of `<main>` [[see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/singleDiv.png)]